### PR TITLE
Adds the Lost Drone

### PR DESCRIPTION
**Pull request by Neerti**
Adds a goodie/baddie for future Points of Interest for the Surface, which is a pod containing a drone which may or may not help you, depending on how RNG feels.
Adds several new lawsets.  Currently there are no law boards for these.
Adds generic 'hey ghosts you wanna do this thing?' prompt and refactors Technomancer Apprentices, Prometheans, posi-drone MMIs, etc.
Adds ability to modify unslaved (no AI) borgs with a law board while they are open.  Each modification takes ten seconds and both of you must remain still.
Adds new 'shocker' baton for the Lost Drone, which works like a less effective stun baton, however it has the unique trait to compel certain simple animals to attack the user, acting as a 'taunt' function.
Adds literal taunt function to simple animal code.
Adds generic 'ghost pods' which let ghosts come back to the round in a different role, based on the pod itself, and will be used for surface content.
Buffs borg shields.  The shield can now sit in one of your three active modules instead of your 'active hand', and it will function.  The shield was also made much more efficient.

### CODE REVIEW

File: code/game/objects/items/weapons/stunbaton.dm [hunk 1]
```diff
@@ -249,4 +249,35 @@
...
+	..(target, user, hit_zone)
+	if(istype(target, /mob/living/simple_animal) && status)
+		var/mob/living/simple_animal/SA = target
+		SA.taunt(user)
+
+// Borg version, for the lost module.
+/obj/item/weapon/melee/baton/shocker/robot
+
+/obj/item/weapon/melee/baton/shocker/robot/attack_self(mob/user)
+	//try to find our power cell
```
Comment (Anewbe) [inline code/game/objects/items/weapons/stunbaton.dm]: Does the base baton not do that?

File: code/game/objects/structures/ghost_pods.dm [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+// This type is triggered by a ghost clicking on it, as opposed to a living player.  A ghost query type isn't needed.
+/obj/structure/ghost_pod/ghost_activated
+	description_info = "A ghost can click on this to return to the round as whatever is contained inside this object."
+
+/obj/structure/ghost_pod/ghost_activated/attack_ghost(var/mob/observer/dead/user)
+	if(used)
+		to_chat(user, "<span class='warning'>Another spirit appears to have gotten to \the [src] before you.  Sorry.</span>")
+		return
+
+	create_occupant(user)
```
Comment (Anewbe) [inline code/game/objects/structures/ghost_pods.dm]: Is there any conformation prompt for this, or does it just suck them in?

File: code/modules/reagents/reagent_containers/borghydro.dm [hunk 3]
```diff
@@ -23,6 +23,9 @@
 /obj/item/weapon/reagent_containers/borghypo/crisis
 	reagent_ids = list("tricordrazine", "inaprovaline", "anti_toxin", "tramadol", "dexalin" ,"spaceacillin")
 
+/obj/item/weapon/reagent_containers/borghypo/lost
+	reagent_ids = list("tricordrazine", "bicaridine", "dexalin", "anti_toxin", "tramadol", "spaceacillin")
```
Comment (Anewbe) [inline code/modules/reagents/reagent_containers/borghydro.dm]: No inaprov?

File: code/game/objects/items/weapons/stunbaton.dm (hunk 1 above)
Comment (Neerti) [reply code/game/objects/items/weapons/stunbaton.dm]: The base baton has its own cell.  The robot version (both this and the regular robot baton) uses the borg's powercell.

File: code/game/objects/structures/ghost_pods.dm (hunk 2 above)
Comment (Neerti) [reply code/game/objects/structures/ghost_pods.dm]: There's no conformation for the ghost click version.

File: code/modules/reagents/reagent_containers/borghydro.dm (hunk 3 above)
Comment (Neerti) [reply code/modules/reagents/reagent_containers/borghydro.dm]: Inaprov seemed weak to me when you have all the other chems but I can add it if desired.

File: code/game/objects/items/weapons/stunbaton.dm (hunk 1 above)
Comment (Anewbe) [reply code/game/objects/items/weapons/stunbaton.dm]: Alrighty.

File: code/modules/reagents/reagent_containers/borghydro.dm (hunk 3 above)
Comment (Anewbe) [reply code/modules/reagents/reagent_containers/borghydro.dm]: Inaprov's got some mildly useful little things it does, might be nice to have it.

File: code/game/objects/structures/ghost_pods.dm (hunk 2 above)
Comment (Anewbe) [reply code/game/objects/structures/ghost_pods.dm]: Might be nice to add one, won't stop me from merging this, though.

**Pull request merged**